### DISCUSSION TITLE
/heal x?

### DISCUSSION THREAD

**Discussion by HiddenStriker** [Pryaxis/TShock#2348] [Ideas]
So I run adventure/rp maps at times and I was wondering since /slap "amount" is possible (it's a great feature for me), would it also be possible to have /heal "amount" so I can choose to heal people for 20-50-100 etc instead of the full healthbar? This sounds liek it would be trivial and pointless for many, but imagine if slap just killed someone instead and we didn't have slap :D It's just a novelty command for sure, but would it be plausible to implement this?

**Comment by moisterrific** [comment]
Sounds doable as an optional parameter to heal, I can work on this if nobody else made a commit already.

**Reply by RussDev7** [reply]
Yeah this is a very easy thing to do.

**Reply by moisterrific** [reply]
this will be added sometime in the future https://github.com/Pryaxis/TShock/pull/2419/commits/72dab02a615a296377929dded12dd4f26a729e69

**Reply by HiddenStriker** [reply]
Thanks so much :D!
